lib/recommended.js

```diff
@@ -66,17 +68,56 @@ module.exports = {
         },
     ],
     ignorePatterns: ["**/build/**", "**/coverage/**", "**/dist/**"],
-    plugins: ["@typescript-eslint", "react"],
+    plugins: ["@typescript-eslint", "react", "@stylistic"],
     env: {
         browser: true,
         node: true,
         es6: true,
     },
     rules: {
+        // @typescript-eslint rules
         "@typescript-eslint/no-unused-vars": [
             "error",
             { argsIgnorePattern: "^_", varsIgnorePattern: "^_" },
         ],
+        "@typescript-eslint/no-unnecessary-condition": "error",
+        "@typescript-eslint/no-floating-promises": "warn",
+
+        // react rules
+        "react/jsx-boolean-value": ["error"],
+        "react/self-closing-comp": ["error"],
+        "react/jsx-no-useless-fragment": ["error", { allowExpressions: true }],
+        "react/jsx-key": ["error"],
+        "react/jsx-sort-props": [
+            "error",
+            {
+                callbacksLast: true,
+                shorthandFirst: true,
+                reservedFirst: true,
+                noSortAlphabetically: true,
```

Comment:
👀 Not really a fan of alphabetical props, you run in to all sorts of situations where you have a component like
```
<Something
	title="title"
	description="description"
/>
```

But the alphabetizing wants the props to be
```
<Something
	description="description"
	title="title"
/>
```

Let me know what you opinion is on this one